Add Kommander 1.1.0 RC 4 Release Notes (#3057)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md

-->

### Kommander 1.1.0 RC 4 - June 26th 2020

| Kubernetes Support | Version |
| ------------------ | ------- |
| **Minimum**        | 1.15.0  |
| **Maximum**        | 1.17.x  |
| **Default**        | 1.17.3  |

#### Features/Improvements

> * Add guideance what to do when cluster deletion failed

Reviewer: ```suggestion
* Add guidance on what to do when cluster deletion fails
```

* Allow default federated addon values to be overriden

> * Attach clusters to Kommander is now considered GA

Reviewer: ```suggestion
* Attaching clusters to Kommander is now considered GA
```

> * Update kubecost web urls

Reviewer: ```suggestion
* Update Kubecost web urls
```

> * Add flag (`--skip-credentials-display`) to Kommander to Konvoy it displaying login information in Logs

Reviewer: ```suggestion
* Add flag (`--skip-credentials-display`) to Kommander so Konvoy does not display login information in logs
```

* Add ability to disable federation of monitoring stack
* Added autogenerated labels to advanced cluster create form
* Improved context handling for multiple contexts when attaching cluster

#### Bug Fixes

> * Kommander Grafana was unavailable after self-attaching host cluster as managed cluster

Reviewer: Is this the detach Kommander bug? Was it only applicable to grafana? I thought all the ingresses stopped working?

Author: not 100% sure  tbh, thats what the JIRA was talking about. I know we fixed UI a while ago (probbaly before 1.0.0), not sure if grafana was missed or UI was the only one we fixed tho

* Fixed a bug where Kommander addon wasnt successfully deploying on Azure
* Fix Kommander to not show unofficially supported versions of Kubernetes by default
* Fix possible data collision bug related to clusters
* Fix a possible crash-loop situation for kubeaddons when cert-manager isnt ready yet
* Fix and Improve LDAP Identity provider handling

> * Fix display of "nothing to report" situations vs acutal errors

Reviewer: ```suggestion
* Fix display of "nothing to report" situations vs actual errors
```
